torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4185,6 +4203,15 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                skipCPUIfNoLapack,
                skipCUDAIfRocm
            ],),
+    OpInfo('einsum',
+           # we need this lambda because SampleInput expects tensor input as the first argument
```

Comment:
Maybe in the future we should support overriding the way sample inputs splat their arguments to more naturally support use cases like this, but this behavior seems rare so let's see if we find a few more examples where it's required